packages/scripts.nix

```diff
@@ -361,4 +361,25 @@
       wait
     '';
   };
+
+  update-terraform-lock-files = writeShellApplication {
+    name = "update-terraform-lock-files";
+    runtimeInputs = with pkgs; [ terraform ];
+    text = ''
+      set -euo pipefail
+      find infra -type f -name "*.tf" -exec dirname {} \; | sort -u | while read -r dir; do
```

Comment:
Compared to the script in constellation we are not filtering out submodules of the same module, but should be ok for now.